mobile/lib/main.dart

```diff
@@ -221,7 +228,34 @@ class _TenTenOneAppState extends State<TenTenOneApp> {
   void initState() {
     super.initState();
 
-    init(context.read<bridge.Config>());
+    final config = context.read<bridge.Config>();
+
+    init(config);
+
+    WidgetsBinding.instance.addPostFrameCallback((_) async {
+      PackageInfo packageInfo = await PackageInfo.fromPlatform();
+
+      final messenger = scaffoldMessengerKey.currentState!;
+
+      try {
+        final response = await http.get(
+          Uri.parse('http://${config.host}:${config.httpPort}/api/version'),
+        );
+
+        final clientVersion = Version.parse(packageInfo.version);
+        final coordinatorVersion = Version.parse(jsonDecode(response.body));
+        FLog.info(text: "Coordinator version: ${coordinatorVersion.toString()}");
+
+        if (coordinatorVersion > clientVersion) {
+          messenger.showSnackBar(SnackBar(
```

Comment:
🔧 I think a snack bar is not the most prominent way of informing the user that she is running an outdated version. How about a popup, that the user has to confirm?

Reply:
I would argue for it, but we don't have to do it in this change.

Reply:
I'm happy to circle back to this tomorrow and get it in!

Reply:
It now shows an alert dialogue